src/commands/start.ts

```diff
@@ -16,10 +16,29 @@ export const start = async (packageVersion: string, apiProvider: ApiProvider, da
     const notes = createNotes(noteInformation, [ details.notes ]);
 
     const harvestApi = apiProvider.getHarvestApi();
+    const targetprocessApi = apiProvider.getTargetprocessApi();
 
     log.info("Starting Harvest timer...");
     try {
         await harvestApi.startTimeEntry(details.projectId, details.taskId, date, notes, details.hours, details.running);
+
+        // Issue #81: If the linked TP entity is a Task *and* its current state is 'Open' or 'Dev ready', set the task to 'In Progress'
+        // Note: It is important to verify current state of Task, otherwise if a 'Done' task with time remaining is moved to 'In Progress',
+        // it may increase the time on a story / feature which the dev / user might not realize as the task has moved to 'In Progress' by this utility
+        // And if the time remaining has not reached 0, the task might not move to 'Done' when the timer is finished
```

Comment:
I think this flags the need for a wiki on this package given it is OSS, we should have things like this documented for visibility we can then reference documentation in the code if it is really necessary - for this case just documenting it should be enough I believe, then we can remove the comments.

Reply:
Added here:

https://github.com/NewOrbit/twotime/wiki/Updating-of-Target-Process-Tasks-via-Timers